locales/ro/translation.json

```diff
@@ -0,0 +1,258 @@
+    "loading-repositories": "Se încarcă depozite ...",
+    "loading-languages": "Se încarcă limbile...",
+    "pick-languages-from-repos": "Te rugăm să alegi cel puțin una dintre limbile disponibile din următoarele depozite: ",
+    "the-selected-repositories": "Depozitele selectate ",
+    "contain-the-following-modules": "conțin următoarele module",
+    "installing-translations": "Se instalează traducerile biblice selectate",
+    "installing-dictionaries": "Se instalează dicționarele selectate",
+    "removing-translations": "Se șterg traducerile biblice",
+    "removing-dictionaries": "Se șterg dicționarele selectate",
+    "it-takes-time-to-install-translation": "Reține că durează ceva timp pentru ca fiecare traducere să fie descărcată și apoi instalată.",
+    "it-takes-time-to-install-dictionary": "Reține că durează ceva timp pentru ca fiecare dicționar să fie descărcat și apoi instalat.",
+    "select-translations-to-be-removed": "Selectează traducerile biblice pe care vrei să le elimini. <br/> <br/> Cele dezactivate din listă sunt traduceri care nu sunt instalate în directorul utilizatorilor.",
+    "select-dictionaries-to-be-removed": "Selectează dicționarele pe care vrei să le elimini. <br/> <br/> Cele dezactivate din listă sunt dicționare care nu sunt instalate în directorul utilizatorului sau dicționare care sunt esențiale pentru funcționalitatea aplicației Ezra Bible.",
+    "installing": "Se instalează",
+    "downloading": "Se descarcă",
+    "part-of-total": "al",
+    "removing": "Se șterge",
+    "installation": "Instalare",
+    "module-lock-info": "Acest modul este protejat de licență și necesită să cumperi o cheie de deblocare de la proprietarul de conținut!",
+    "unlock-key": "Cheia de deblocare",
+    "enter-unlock-key": "{{moduleId}} este protejat prin licență. Te rog introdu Cheia de deblocare!",
+    "unlock-failed": "Deblocare eșuată! Te rog introdu Cheia de deblocare corectă."
+  },
+  "module-sync": {
+    "module-sync-header": "Se instalează module adiționale"
+  },
+  "general": {
+    "chapter-verse-separator": ":",
+    "module-application-info": "Informații despre module și aplicație",
+    "application-info": "Informații despre aplicație",
+    "sword-module-description": "Descriere modul SWORD",
+    "sword-module-details": "Detalii modul SWORD",
+    "sword-module-info": "Informații modul SWORD",
+    "sword-unlock-info": "Informații deblocare SWORD",
+    "application-version": "Versiune aplicație",
+    "git-commit": "Git commit",
+    "sword-version": "versiune SWORD",
+    "chromium-version": "versiune Chromium",
+    "database-path": "cale Bază de date",
+    "config-file-path": "cale Configurare",
+    "module-name": "Nume",
+    "module-version": "Versiune",
+    "module-language": "Limbă",
+    "module-license": "Licență",
+    "module-strongs": "Concordanță",
+    "module-hebrew-strongs-dict": "Concordanță Ebraică",
```

Comment:
The term Strong's is a unique concept, I suggest to keep that

Reply:
Isn't  '_Strong's_' concept just an index of words to be explained? If you would like to keep the author name in this (James Strong) then what about '**Concordanța lui Strong**' = '**Strong's Concordance**' otherwise i will leave this untranslated as it has no understanding in Romanian.

Reply:
Yes to what you suggested. The specific numbering system is from this guy named Strong and it is unique, not just a generic concept.